### ISSUE TITLE
Exercise 5, Task 3, Step 8 - Editing functions in the Azure portal is not supported for Linux Consumption Function Apps.

### ISSUE THREAD

**Issue by ewittrock-uwm** [microsoft/MCW-Azure-Synapse-Analytics-and-AI#43]
Exercise 5, Task 3, Step 8 says to select GetInvoiceData from the Functions listing in the Azure Portal.  However, the portal does not list any functions on that screen and displays the following warning near the top of the page:  "Editing functions in the Azure portal is not supported for Linux Consumption Function Apps".  This means step 9 cannot be completed either (obtaining the function URL for later reference).  

Is there an alternative method to obtain the function URL?

**Comment by ewittrock-uwm** [comment]
I determined that the Function was not being properly deployed to Azure.  On my system I needed to install the Azure Functions Core Tools ([https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=windows%2Ccsharp%2Cbash]) .  I determined this by following the basic Azure Function tutorial on the Azure Functions Extension for Visual Studio Code page in VS Code.  

After installing the Tools, the function deployed properly and I was able to get the URL in the portal.   It would be helpful to add this as a potential requirement in the Before the HOL instructions.  

Thanks!
